pyiron/base/database/filetable.py

```diff
@@ -0,0 +1,365 @@
+import numpy as np
+import os
+import pandas
+import datetime
+import h5io
+import sys
+from six import with_metaclass
+from pyfileindex import PyFileIndex
```

Comment:
URGENT: This creates ModuleNotFoundError: No module named 'pyfileindex'

Reply:
Should be fixed now